build.gradle

```diff
@@ -6,26 +6,26 @@ ext {
     minSdkVersion = 16
     targetSdkVersion = 27
     zendeskSdkVersion = "2.2.1"
-    zopimSdkVersion = "1.4.0"
+    zopimSdkVersion = "1.4.1"
```

Comment:
Actually we should probably update `zendeskSdkVersion` and `zopimSdkVersion` to `supportSdkVersion` and `chatSdkVersion`.

Reply:
Will do!